pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -972,4 +974,11 @@ public static class Range {
   public static class IdealState {
     public static final String HYBRID_TABLE_TIME_BOUNDARY = "HYBRID_TABLE_TIME_BOUNDARY";
   }
+
+  public static class RewriterConstants {
+    public static final String PARENT_AGGREGATION_NAME_PREFIX = "pinotparentaggregation";
+    public static final String CHILD_AGGREGATION_NAME_PREFIX = "pinotchildaggregation";
```

Comment:
intentionally kept this way as otherwise the ArgMinMaxRewriterTest will not be able to test. When the function is parsed everything gets rewritten to small case